USAGE.md

```diff
@@ -8,100 +8,281 @@ Baritone commands can also by default be typed in the chatbox. However if you ma
-- `repack` to re-cache the chunks around you.
-- `gc` to call `System.gc()` which may free up some memory.
-- `render` to fix glitched chunk rendering without having to reload all of them.
-- `reloadall` to reload Baritone's world cache or `saveall` to save Baritone's world cache.
-- `find` to search through Baritone's cache and attempt to find the location of the block.
-- `surface` or `top` to tell Baritone to head towards the closest surface-like area, this can be the surface or highest available air space.
-- `version` to get the version of Baritone you're running
-- `damn` daniel
-
-All the settings and documentation are <a href="https://github.com/cabaletta/baritone/blob/master/src/api/java/baritone/api/Settings.java">here</a>. If you find HTML easier to read than Javadoc, you can look <a href="https://baritone.leijurv.com/baritone/api/Settings.html#field.detail">here</a>.
-
-There are about a hundred settings, but here are some fun / interesting / important ones that you might want to look at changing in normal usage of Baritone. The documentation for each can be found at the above links.
-- `allowBreak`
-- `allowSprint`
-- `allowPlace`
-- `allowParkour`
-- `allowParkourPlace`
-- `blockPlacementPenalty`
-- `renderCachedChunks` (and `cachedChunksOpacity`) <-- very fun but you need a beefy computer
-- `avoidance` (avoidance of mobs / mob spawners)
-- `legitMine`
-- `followRadius`
-- `backfill` (fill in tunnels behind you)
-- `buildInLayers`
-- `buildRepeatDistance` and `buildRepeatDirection`
-- `worldExploringChunkOffset`
-- `acceptableThrowawayItems`
-- `blocksToAvoidBreaking`
-- `mineScanDroppedItems`
-- `allowDiagonalAscend`
-
-
-
-
-# Troubleshooting / common issues
+
+
+
+# Process Management
+
+Baritone uses a process system where only one main process can be active at a time. Available processes include:
+
+- **Builder Process** - Building schematics and structures
+- **Mine Process** - Mining specific blocks or quantities  
+- **Farm Process** - Automatic farming operations
+- **Follow Process** - Following players, entities, or picking up items
+- **Explore Process** - World exploration and chunk discovery
+- **Elytra Process** - Automated elytra flying in the Nether
+- **Custom Goal Process** - Manual goal setting and pathfinding
+- **Get To Block Process** - Pathfinding to specific block types
+
+Use `proc` to see which process is currently active and get information about its state.
+
+# Troubleshooting / Common Issues
 
 ## Why doesn't Baritone respond to any of my chat commands?
-This could be one of many things.
+This could be several things:
+
+1. **Check Installation**: Make sure Baritone is installed. It should create a `baritone` folder in your Minecraft directory.
+
+2. **Check Prefix Settings**: Verify you're using the correct prefix:
+   - Default prefix is `#` (e.g., `#help`)
+   - Some clients like Impact use `.b` as an alias
```

Comment:
Impact is ancient. Better just write something generic like "some clients add an alternative alias" or "some clients have alternative commands to control Baritone" or similar.